Gemfile.lock

```diff
@@ -152,7 +152,7 @@ GEM
       ffi (~> 1.0)
     regexp_parser (1.6.0)
     ruby_dep (1.5.0)
-    rubyzip (2.0.0)
+    rubyzip (2.2.0)
```

Comment:
### [rubyzip](http://github.com/rubyzip/rubyzip)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.0.0 → 2.2.0

[_[change-log](https://github.com/rubyzip/rubyzip/blob/v2.2.0/Changelog.md), [source-code](https://github.com/rubyzip/rubyzip/tree/v2.2.0)_]





<details>
<summary>Commits</summary>

A change of **64** commits. See the full changes on [the compare page](https://github.com/rubyzip/rubyzip/compare/v2.0.0...v2.2.0).

These are the first 10 commits:
- (506d557) [StreamableStream now uses the OS temp directory.](https://github.com/rubyzip/rubyzip/commit/506d557edc29144c6f1e3110b4b99043232c3eaf)
- (e871842) [Require 'tmpdir' only in the tests.](https://github.com/rubyzip/rubyzip/commit/e87184200ab92220876577d312ce78354adf6bbf)
- (a9a313a) [Merge pull request #411 from hainesr/tempfiles](https://github.com/rubyzip/rubyzip/commit/a9a313a386b5c5e7cc900a7165f8107dfa019d65)
- (34d2074) [Update changelog for #411](https://github.com/rubyzip/rubyzip/commit/34d2074ecbf6ef8557c29efc03a585c309be5c4b)
- (340379f) [Use `warn` instead of `puts` for messages from the library c…](https://github.com/rubyzip/rubyzip/commit/340379f0808e9cc18df0248aeec07bdb95016faa)
- (935a4f3) [Make warning messages consistent.](https://github.com/rubyzip/rubyzip/commit/935a4f31a2fc781b2c8b41cadb42ab9fd85d2a6e)
- (ccabd94) [Update tests to check error messages.](https://github.com/rubyzip/rubyzip/commit/ccabd94e42dd1f29cf0d4f9897926aadd854e98b)
- (f58e380) [Add the `options` parameter to `Zip::File.open`.](https://github.com/rubyzip/rubyzip/commit/f58e38012cf4d7493f70ffdc4e9a583d94f72ad5)
- (3fea995) [Update comment for `Zip::File.open`.](https://github.com/rubyzip/rubyzip/commit/3fea9958c2da24d16b20f545d982ad0c0a3d7f57)
- (15a54b4) [Merge pull request #416 from hainesr/warnings](https://github.com/rubyzip/rubyzip/commit/15a54b4c2b986c56430dde6c441312c7b38a2093)


</details>